### PR TITLE
added logic to handle previously unhandled exception

### PR DESCRIPTION
**Pull request by bsmithvt**

### CODE REVIEW

File: Fabric.Identity.API/Quickstart/Account/AccountController.cs [hunk 1]
```diff
@@ -262,12 +262,21 @@ public async Task<IActionResult> ExternalLoginCallback(string returnUrl)
...
+            try
+            {
+                //issue authentication cookie for user
+                user = await _userLoginManager.UserLogin(
+                    claimInformation.Provider,
+                    _claimsService.GetEffectiveUserId(claimInformation),
+                    claimInformation.Claims,
+                    claimInformation?.ClientId);
+            }
+            catch (Exception e)
```
Comment (michaelvidal24) [inline Fabric.Identity.API/Quickstart/Account/AccountController.cs]: Should we be catching a generic exception here? Seems like we might cover up some issues that we would want to know about and fix.

Comment (bsmithvt) [reply Fabric.Identity.API/Quickstart/Account/AccountController.cs]: Yes, this is for that bug. Are there any cases where we want the error to be surfaced to the end user? I've also added more details to the log - I just noticed the actual details of the SqlException are due to data being potentially truncated.

Comment (michaelvidal24) [reply Fabric.Identity.API/Quickstart/Account/AccountController.cs]: I don't think we can just swallow it, we need to figure out how to handle this exceptional case so users can successfully authenticate.

Comment (bsmithvt) [reply Fabric.Identity.API/Quickstart/Account/AccountController.cs]: Are you saying they should be able to authenticate in this scenario? Why would I be able to authenticate with my HC account if it's not whitelisted?

Comment (michaelvidal24) [reply Fabric.Identity.API/Quickstart/Account/AccountController.cs]: I believe the HC tenant is a whitelisted tenant in the exploratory test environment.

**Pull request closed without merging**